collections/_article/dev-snapshot-godot-4-6-dev-3.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Dev snapshot: Godot 4.6 dev 3"
+excerpt: Rocking a new look!
+categories: [pre-release]
+author: Thaddeus Crews
+image: /storage/blog/covers/dev-snapshot-godot-4-6-dev-3.jpg
+image_caption_title: My Card Is Better Than Your Card!
+image_caption_description: A game by Utu Studios
+date: 2025-11-04 12:00:00
+---
+
+Our third development snapshot of the 4.6 release cycle is upon us, and now the floodgates are well and truly open. The enhancements are bigger than ever, with several of them overhauling the look of the editor itself entirely! Big features necessitate big changes, which means the potential for big bugs and regressions; early reports and tests remain crucial catch these issues as soon as possible.
+
+Please consider [supporting the project financially](#support), if you are able. Godot is maintained by the efforts of volunteers and a small team of paid contributors. Your donations go towards sponsoring their work and ensuring they can dedicate their undivided attention to the needs of the project.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also try the [**Web editor**](https://editor.godotengine.org/releases/4.6.dev3/), the [**XR editor**](https://www.meta.com/s/3yJ7i8kop), or the [**Android editor**](https://play.google.com/store/apps/details?id=org.godotengine.editor.v4) for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+---
+
+*The cover illustration is from* [**My Card Is Better Than Your Card!**](https://store.steampowered.com/app/3617620/My_Card_Is_Better_Than_Your_Card/?curator_clanid=41324400), *an idle game where you build and optimize your computer's system from the ground up. You can get the game or try the demo [Steam](https://store.steampowered.com/app/3617620/My_Card_Is_Better_Than_Your_Card/?curator_clanid=41324400), and follow the developers on [YouTube](https://www.youtube.com/@UtuStudios) or [Bluesky](https://bsky.app/profile/mycard.utustudios.com).*
```

Comment:
```suggestion
*The cover illustration is from* [**My Card Is Better Than Your Card!**](https://store.steampowered.com/app/3617620/My_Card_Is_Better_Than_Your_Card/?curator_clanid=41324400), *an idle game where you build and optimize your computer's system from the ground up. You can get the game or try the demo on [Steam](https://store.steampowered.com/app/3617620/My_Card_Is_Better_Than_Your_Card/?curator_clanid=41324400), and follow the developers on [YouTube](https://www.youtube.com/@UtuStudios) or [Bluesky](https://bsky.app/profile/mycard.utustudios.com).*
```
I've only looked briefly at the Steam page, and this doesn't appear to be "an idle game where you build and optimize your computer's system from the ground up"...?

Reply:
It's funny cos I thought maybe its name was meant in the sense of "My Graphics Card Is Better Than Your Graphics Card"!